hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/common/CachingBlockManager.java

```diff
@@ -354,7 +355,7 @@ private void readBlock(BufferData data, boolean isPrefetch, BufferData.State...
     }
 
     @Override
-    public Void applyE() {
+    public Void get() {
```

Comment:
we need to think what we want to do here.

1. we would want errors to be counted and included in stream stats
1. but we've seen abfs apps flooded with stack traces when a transient network error breaks every prefetch

* its good to be aware of transient errors, but not worry too much until the final reaqd.

created https://issues.apache.org/jira/browse/HADOOP-18185